### PR TITLE
Updating .mailmap

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@schiesbn

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by DeepDiver1975** [comment]
@LukasReschke @nickvergessen @karlitschek please review - thx

File: apps/files/ajax/move.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * @author Bjoern Schiessle <schiessle@owncloud.com>
+ * @author Björn Schießle <schiessle@owncloud.com>
  * @author Frank Karlitschek <frank@owncloud.org>
- * @author Georg Ehrke <georg@owncloud.com>
+ * @author Georg Ehrke <georg@ownCloud.com>
```
Comment (nickvergessen) [inline apps/files/ajax/move.php]: better fix the email?

File: apps/files_external/ajax/google.php [hunk 2]
```diff
@@ -4,7 +4,6 @@
  * @author Christopher Schäpers <kondou@ts.unde.re>
  * @author Jörn Friedrich Dreyer <jfd@butonic.de>
  * @author Lukas Reschke <lukas@owncloud.com>
- * @author Michael Gapczynski <gapczynskim@gmail.com>
  * @author Michael Gapczynski <GapczynskiM@gmail.com>
```
Comment (nickvergessen) [inline apps/files_external/ajax/google.php]: Should we prefer the lowercase version here aswell?

**Comment by nickvergessen** [comment]
Can you fix my email to `@owncloud.com` too, I fixed my git config just now

**Comment by DeepDiver1975** [comment]
:-1: on hold as we detected issues - I'll reuse this branch to fix open issues - THX

**Comment by DeepDiver1975** [comment]
@MorrisJobke @nickvergessen @karlitschek mind reviewing? THX

File: core/command/background/ajax.php [hunk 3]
```diff
@@ -1,27 +1,23 @@
 <?php
 /**
-* The MIT License (MIT)
```
Comment (DeepDiver1975) [inline core/command/background/ajax.php]: @karlitschek this is the case I was revering to in my email - from my understanding we should not overwrite such license headers.

**Comment by karlitschek** [comment]
:+1:

**Comment by nickvergessen** [comment]
@karlitschek you +1-ed something that has an open question directed at you.
Would be nice, if you take a look again and reply to the question ;)

**Comment by karlitschek** [comment]
O.K. sorry. This was only a response to the "please review" questions :-)

**Comment by MorrisJobke** [comment]
:+1:

@karlitschek Once you answered the question above we can merge this. ;)

**Comment by DeepDiver1975** [comment]
:-1: we are not done - we have to make adoptions to the license header and respect existing MIT headers.

**Comment by MorrisJobke** [comment]
> we are not done - we have to make adoptions to the license header and respect existing MIT headers.

Okay. I thought it is just this one file with the MIT header

**Comment by DeepDiver1975** [comment]
@karlitschek files containing the MIT license are ignored for the time being

**Comment by MorrisJobke** [comment]
rebase needed ;)

**Comment by DeepDiver1975** [comment]
> rebase needed ;)

nooo432

**Comment by DeepDiver1975** [comment]
@MorrisJobke @karlitschek merge time :question:

**Comment by karlitschek** [comment]
:+1:

**Comment by MorrisJobke** [comment]
* apps/files_encryption/ajax/*
* lib/private/defaults.php
* lib/private/hook.php
* lib/private/preview.php
* lib/private/util.php


That should be all the problematic files

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**